### PR TITLE
add healing enemy

### PR DESCRIPTION
**Pull request by RuslanKalashnikov**

### CODE REVIEW

File: code/game/controllers/enemies.py [hunk 1]
```diff
@@ -47,11 +47,12 @@ def _get_enemy(self) -> EnemyController:
 
         if self._start_time + 2 * LEVEL_INTERVAL < time():
             available_enemy_types.append(LightEnemyController)
-            spawn_probabilities = [60, 20, 20]
+            available_enemy_types.append(HealEnemyController)
```
Comment (kuanysh-beisembayev) [inline code/game/controllers/enemies.py]: давай использовать `extend` тут

File: code/game/controllers/planet.py [hunk 2]
```diff
@@ -13,6 +13,9 @@ def get_damage(self, damage: int) -> None:
         if self._health == 0:
             raise GameOver
 
+    def get_heal(self, heal: int) -> None:
+        self._health = min(self._health + heal, 100)
```
Comment (kuanysh-beisembayev) [inline code/game/controllers/planet.py]:
```suggestion
        self._health = min(self._health + heal, self.MAX_HEALTH)
```

Comment (RuslanKalashnikov) [reply code/game/controllers/planet.py]: sry на автомате проставил

File: tests/game/controllers/test_planet.py [hunk 3]
```diff
@@ -22,3 +22,21 @@ def test_get_damage_full() -> None:
...
+
+def test_get_heal() -> None:
+    controller = PlanetController()
+
+    controller.get_damage(30)
+    assert controller._health == 70
+
+    controller.get_heal(10)
+
+    assert controller._health == 80
```
Comment (kuanysh-beisembayev) [inline tests/game/controllers/test_planet.py]: вот тут как раз нужно parametrize использовать. И давай corner-case проверим тоже

**Review by kuanysh-beisembayev — APPROVED** [review]
